bff/pkg/server/server.go

```diff
@@ -319,67 +339,72 @@ func metrics_handler(route_configs map[string]RouteConfig, dbIdentifiers []strin
 	})
 }
 
-func activity_handler(metrics_service metrics.Service) http.HandlerFunc {
+type ActivityParams struct {
+	DbIdentifier string `validate:"required,dbIdentifier,max=50"`
+	DatabaseList string `validate:"required,databaseList,max=50"`
```

Comment:
For databaseList, the max length of 50 is too low, as multiple databases can be selected and each of them might have a long name. I don't think that we should put a max length, but a max length (50) for each db name is fine.